### DISCUSSION TITLE
OSGi support in jPOS 3.x removed

### DISCUSSION THREAD

**Discussion by ar** [jpos/jPOS#548] [General]
OSGi has been discontinued in 3.0. This decision was driven by the observation that the feature never gained significant traction within our community. Should the community requirements change in the future, and we see a renewed demand for OSGi support, we would certainly consider re-enabling it.

We encourage open discussion and welcome any feedback on this matter.

**Comment by Bilal2547** [comment]
how to remove other group members from our group
